js/views/chatview.js

```diff
@@ -93,9 +93,10 @@
 				}, params));
 			}
 
+			var guestNick = OCA.SpreedMe.app._localStorageModel.get('nick');
 			return this._addCommentTemplate(_.extend({
 				actorId: '',
-				actorDisplayName: t('spreed', 'You'),
+				actorDisplayName: guestNick ? guestNick : t('spreed', 'You'),
```

Comment:
Should have the Text edit view here too, so you can change your guest name just above the comment box. Can you do that @danxuliu

Reply:
Also currently it does not update when you set/change your name